include/irr/asset/ICPUBuffer.h

```diff
@@ -62,7 +62,8 @@ class ICPUBuffer : public asset::IBuffer, public asset::IAsset
 
         virtual void convertToDummyObject(uint32_t referenceLevelsBelowToConvert=0u) override
         {
-            _IRR_ALIGNED_FREE(data);
+            if (data)
+                _IRR_ALIGNED_FREE(data);
             data = nullptr;
             size = 0ull;
             isDummyObjectForCacheAliasing = true;

```

Comment:
can you factor this out to a common non-virtual inline function, otherwise the custom ICPUBuffers will still keep intact `size` and `isDummyObjectForCacheAliasing`

Reply:
done in todays commit